.rubocop.yml

```diff
@@ -1,5 +1,8 @@
 inherit_from: ./config/default.yml
 
+require:
+  - rubocop-performance
```

Comment:
Are some of the renamed checks in Rubocop Performance, or is this a separate change?

Reply:
hmm, we are using some Performance cops in the `.rubocop.yml` of rubocop-github itself, and I got errors when `rubocop-performance` is not required